src/content/en/tools/lighthouse/http-redirects-to-https.md

```diff
@@ -0,0 +1,35 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site redirects HTTP traffic to HTTPS" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-20 #}
+{# wf_published_on: 2016-09-20 #}
+
+# Site redirects HTTP traffic to HTTPS  {: .page-title }
+
+## Why the audit is important {: #why }
+
+All sites should be protected with HTTPS. See the following Lighthouse doc to
+learn why: [Site is on HTTPS](https).
+
+Once you've got HTTPS set up, you need to make sure that all unsecure HTTP
+traffic to your site is redirected to HTTPS. 
+
+## How to pass the audit {: #how }
+
+1. Use canonical links in the `head` of your HTML to help search engines figure
+   out the best way to get to the page.
+
+       <link rel="canonical" href="https://example.com"/>
+
+2. Configure your server to redirect HTTP traffic to HTTPS. See your server's
+   documentation to figure out the best way to do this.
+
```

Comment:
Migrating to HTTPS is very complicated, but this audit pertains specifically to redirecting HTTP to HTTPS, which is not that hard to set up.